DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+        to target cell
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C115542
+    validation_result:
+      description: Result confirming successful differentiation to target cell type
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C16237
+    markers:
+      description: A list of markers used to enrich for or against certain cells
+      type: string
+      format: alphanumerical
+      ontology_link: https://bioportal.bioontology.org/ontologies/LOINC?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FLNC%2FLP7783-6
+    minimum_size:
+      description: Minimum cell or organelle size passing selection, in microns
+      type: integer
+      format: "\xB5m"
+      upper_value: 10000.0
+      lower_value: 0.01
+      ontology_link: https://bioportal.bioontology.org/ontologies/ONTODT?p=classes&conceptid=http%3A%2F%2Fontodm.com%2FOntoDT%23OntoDT_844136
+    maximum_size:
+      description: Maximum cell or organelle size passing selection, in microns
+      type: integer
+      format: "\xB5m"
+      upper_value: 10000.0
+      lower_value: 0.01
+      ontology_link: https://bioportal.bioontology.org/ontologies/ONTODT?p=classes&conceptid=http%3A%2F%2Fontodm.com%2FOntoDT%23OntoDT_844136
+    method_induction:
+      description: Induction method applied to primary cell culture to induce pluripotent
+        stem cell generation
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C61367
+    reprogramming_factors:
+      description: Reprogramming factors added to primary cell culture to induce pluripotency
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/MESH?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FMESH%2FD065150
+    ipsc_induction_kit:
+      description: Kit used to induce pluripotent stem cell generation
+      type: string
+      format: text
+    pluripotency_test:
+      description: Description of how pluripotency was tested in induced pluripotent
+        stem cells
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/PLOSTHES?p=classes&conceptid=http%3A%2F%2Flocalhost%2Fplosthes.2017-1%236180
+    percent_pluripotency:
+      description: Percent of iPSCs that passed the pluripotency test
+      type: integer
+      format: text
+    pluripotency_vector_removed:
+      description: Whether a viral vector was removed after induction
+      type: boolean
+      format: boolean
+      options:
```

Comment:
@ishitasati11 Use `enum` instead of `options` to specify list of possible values.
https://swagger.io/docs/specification/data-models/enums/

Reply:
Also values should be without quotes